code-challenges.test.js

```diff
@@ -22,13 +24,36 @@ const fibonacciLength2 = 10
 
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
+// Pseudo code: have to have the array store the fibonnacci sequence starting with the first two indexes of the array [1,1] in the for loop i had the index -2. if 1 was less than 2 it would add 1. i then added both of my firstTwo including the indexes. once the array has gone through all indexes you will need it to return. then i console.log logged both fibonacci lengths
+
+describe( "greaterThanTwo",() => {
+  it ("takes in a number (greater than 2) and returns the array with the fibonacci squence" , () => {
+     expect (greaterThanTwo(fibonacciLength1)).toEqual([1, 1, 2, 3, 5, 8])
+     expect (greaterThanTwo(fibonacciLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+     
+  })
+})
+
+const greaterThanTwo = (number) => {
+  const firstTwo = [1,1]
+  for(let i = 2; i<number; i++ )
+  firstTwo[i] = firstTwo[i-1] + firstTwo[i-2]
+return firstTwo
+}
+console.log(greaterThanTwo(fibonacciLength1))
+console.log(greaterThanTwo(fibonacciLength2))
```

Comment:
Nice job! Remember that the test will call your function. The console.log may actually cause issues in your code.